draft-ietf-quic-transport.md

```diff
@@ -5018,10 +5034,16 @@ sequence number, or if a sequence number is used for different connection
 IDs, the endpoint MAY treat that receipt as a connection error of type
 PROTOCOL_VIOLATION.
 
+If the Retire Prior To field is present it is a request for the peer to
+immediately retire all connection IDs with a sequence number less than the
+specified value.  For more information see {{retiring-cids}}.  This field
+should only ever increase, though reordering and retransmission of older packets
```

Comment:
There's lots of vague 2119-lite language here.  The language on flow control can be used here: 

> A receiver cannot renege on an advertisement however. That is, once a receiver advertises an offset, it MAY advertise a smaller offset, but this has no effect.
> [...]
> A sender MUST ignore any MAX_STREAM_DATA or MAX_DATA frames that do not increase flow control limits.